docs/ceph-csi-upgrade.md

```diff
@@ -24,6 +25,7 @@
         - [4.2 Update the RBD Nodeplugin daemonset](#42-update-the-rbd-nodeplugin-daemonset)
         - [4.3 Manual deletion of RBD Nodeplugin daemonset pods](#43-manual-deletion-of-rbd-nodeplugin-daemonset-pods)
     - [Handling node reboot hangs due to existing network mounts](#handling-node-reboot-hangs-due-to-existing-network-mounts)
+    - [CSI Sidecar containers considerations](#csi-sidecar-containers-considerations)
```

Comment:
link is not working, need to be changed to `[CSI Sidecar containers consideration](#csi-sidecar-containers-consideration)`

Reply:
fixed